validator/fork/config.json

```diff
@@ -0,0 +1,66 @@
+{
+  "Impl": "mesa",
+  "HTTP": {
+    "Port": "8080",
+    "RateLimInterval": "1s",
+    "MaxRequestPerInterval": 10
+  },
+  "Gateway": {
+    "ExternalURIPrefix": "http://localhost:8080",
+    "AnimationRendererURI": "https://render.tableland.xyz"
+  },
+  "DB": {
+    "Port": "5432"
+  },
+  "TableConstraints": {
+    "MaxRowCount": 500000
+  },
+  "QueryConstraints": {
+    "MaxWriteQuerySize": 35000,
+    "MaxReadQuerySize": 35000
+  },
+  "Metrics": {
+    "Port": "9090"
+  },
+  "Log": {
+    "Human": true,
+    "Debug": true
+  },
+  "Analytics": {
+    "FetchExtraBlockInfo": true
+  },
+  "Backup": {
+    "Enabled": false
+  },
+  "TelemetryPublisher": {
+    "Enabled": false
+  },
+  "Chains": [
+    {
+      "Name": "Local Hardhat",
+      "ChainID": 1,
```

Comment:
As noted elsewhere, the hardhat chainId is now `1`.  I haven't really explored the implications of this, but I would assume that dev's will have to reconfigure their a browser wallet to handle this.